test/components/dataset-summary/row.spec.js

```diff
@@ -44,4 +44,25 @@ describe('Dataset summary row', () => {
       });
     });
   });
+
+  it('should show help text when there is no properties', async () => {
+    const dataset = testData.formDraftDatasetDiffs.createPast(1, { isNew: true, properties: [] }).last();
+    const component = mount(Row, {
+      props: { dataset, projectId: 1 },
+      container: {
+        router: mockRouter('/')
+      }
+    });
+    component.getComponent(RouterLinkStub).props().to.should.be.equal(`/projects/1/datasets/${dataset.name}`);
```

Comment:
I think this test will fail once merged to `master`, now that #841 is merged. You could try rebasing, or merge `master` into this branch?